src/main/java/mate/controller/car/CreateCarController.java

```diff
@@ -0,0 +1,53 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.DriverService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/cars/add")
+public class CreateCarController extends HttpServlet {
```

Comment:
Better use the same word for the same action everywhere. **Add** everywhere or **create** everywhere.
**For all controllers.**
```suggestion
@WebServlet(urlPatterns = "/cars/add")
public class AddCarController extends HttpServlet {
```